### DISCUSSION TITLE
FontAwesome Iconos no reconocidos?

### DISCUSSION THREAD

**Discussion by MesachVenegas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#348] [Q&A] [answered]
Hola que tal estoy iniciando con la landing page de vacunación, e intento usar FontAwesome para los iconos pero algunos no los reconoce y eso que los estoy tomando directamente de su sitio web
![image](https://user-images.githubusercontent.com/99062079/157828132-d866e3f6-c730-4c3f-adef-20649d38b974.png)

me parece un signo como de alerta en vez del icono del lapiz que  trato de usar

![image](https://user-images.githubusercontent.com/99062079/157828249-2821edcd-e123-4f57-8b8a-9a306516d0e6.png)

![image](https://user-images.githubusercontent.com/99062079/157828293-9d20bf1e-87cc-42c7-836e-71c9cc568add.png)

alguno le ha pasado y sabe a que se deba, estoy usando FontAwesome V6.0

![image](https://user-images.githubusercontent.com/99062079/157828377-b05ac3ad-72d5-4105-8de9-1423d053c8f2.png)

**Comment by AngelCruzO** [comment, marked as answer]
No te deja usarlos porque son de la versión Pro, solo puedes ocupar los free

**Reply by MesachVenegas** [reply]
Lol, nunca me había topado con eso 🤣🤣, gracias

**Comment by Jimdrer** [comment]
Tengo una pregunta, porque tienes 2 páginas CSS?

**Reply by Ingrzs** [reply]
Pues en una hoja personaliza los grid y en otro los css, tal ves no quiere tener mucho código en una.

**Reply by MesachVenegas** [reply]
> Pues en una hoja personaliza los grid y en otro los css, tal ves no quiere tener mucho código en una.

En efecto, aún que al final me decidí por dejarlo todo en una sola hoja de estilos 😁

**Comment by doguedogue** [comment]
Usa los gratuitos ejemplo fa-pen

**Reply by MesachVenegas** [reply]
Gracias eso are
